src/main/java/it/pagopa/interop/probing/eservice/registry/updater/util/EserviceState.java

```diff
@@ -6,9 +6,11 @@
 
 public enum EserviceState {
 
-	ACTIVE("ACTIVE"),
+	ONLINE("ONLINE"),
 
-	INACTIVE("INACTIVE");
+	OFFLINE("OFFLINE"),
+
+	N_D("N/D");
```

Comment:
`n/d` is a calculated value, it will not be stored

Reply:
done